explorer/app/components/StatusBadge/statusBadge.tsx

```diff
@@ -0,0 +1,32 @@
+// Core dependencies
+import { Chip } from "@mui/material";
+import React from "react";
+
+export enum STATUS {
```

Comment:
Can you create a single config object out of `STATUS`, `BADGE_COLOR` and `BADGE_LABEL`?